ranger/ext/img_display.py

```diff
@@ -17,23 +17,43 @@
 import fcntl
 import imghdr
 import os
+import shlex
 import struct
 import sys
 import warnings
 import json
+import mmap
+import shutil
 import threading
-from subprocess import Popen, PIPE
-from collections import defaultdict
+from subprocess import Popen, PIPE, check_call, check_output, CalledProcessError
+from collections import defaultdict, namedtuple
 
 import termios
 from contextlib import contextmanager
 import codecs
-from tempfile import NamedTemporaryFile
+from tempfile import TemporaryFile, NamedTemporaryFile
 
 from ranger import PY3
 from ranger.core.shared import FileManagerAware, SettingsAware
 from ranger.ext.popen23 import Popen23
 
+
+def _which(cmd):
```

Comment:
Oughta be moved to another package

Reply:
Could go in `ranger/ext` I guess. Though it does seem like `get_executables` may have a very similar purpose, maybe we should just use that?